lib/generators/rolemodel/saas/devise/templates/app/views/devise/invitations/edit.html.slim

```diff
@@ -0,0 +1,28 @@
+.page__header-container
+  nav.page__header
+    .nav-page__header-title-group
+      h1.nav-page__header-title= t 'devise.invitations.edit.header'
+
+.user-form
+  = form_for resource, as: resource_name, url: invitation_path(resource_name), html: { method: :put } do |f|
+    = devise_error_messages!
+    = f.hidden_field :invitation_token, readonly: true
+
+    - if resource.ninja_passport_id
```

Comment:
Specific to NM

Reply:
absolutely - again probably need to review each file pulled in, which is deeper than what I did